### ISSUE TITLE
[sbt 1.5.0-M2] Can't use `inputFileChanges` nor `inputFiles` without warnings

### ISSUE THREAD

**Issue by rdesgroppes** [sbt/sbt#6330]
## steps

Have a source code generator using either `inputFileChanges` and/or `inputFiles`.

## problem

```
[error] /path/to/project/CodeGenerator.scala:37:33: method in in class Scope is deprecated (since 1.5.0): `in` is deprecated; migrate to slash syntax - https://www.scala-sbt.org/1.x/docs/Migrating-from-sbt-013x.html#slash
[error]         (config / genCode).inputFiles,
[error]                            ^
[error] /path/to/project/CodeGenerator.scala:36:33: method in in class Scope is deprecated (since 1.5.0): `in` is deprecated; migrate to slash syntax - https://www.scala-sbt.org/1.x/docs/Migrating-from-sbt-013x.html#slash
[error]         (config / genCode).inputFileChanges,
[error]                            ^
```

## expectation

Either no warning (treated as errors in my case), or a warning telling what to do instead.

## notes

https://www.scala-sbt.org/1.x/docs/Migrating-from-sbt-013x.html#slash doesn't give any hint on how to replace `inputFiles` and `inputFileChanges` macros with the slash syntax.

**Comment by eed3si9n** [comment]
@rdesgroppes Thanks for the report!

**Comment by eatkins** [comment]
I was recently wondering if any builds actually use these apis and was glad to see that at least someone is! I pushed a fix for this issue in #6373.
